run.sh

```diff
@@ -2,6 +2,11 @@
 MODE=${MODE:="default"}
 RS_MODE=${RS_MODE:=$MODE}
 
+envsubst '$${RS_RTMP_PORT}' < /restreamer/conf/nginx.conf.template > /restreamer/conf/nginx.conf
+envsubst '$${RS_RTMP_PORT}' < /restreamer/conf/nginx_ssl.conf.template > /restreamer/conf/nginx_ssl.conf
+envsubst '$${RS_RTMP_PORT}' < /restreamer/conf/live.json.template > /restreamer/conf/live.json
+envsubst '$${RS_RTMP_PORT}' < /restreamer/conf/live-rpi.json.template > /restreamer/conf/live-rpi.json
```

Comment:
this is where the magic happens - the rest of the changes are basically replacing 1935 with the env variable and setting a default to 1935

Reply:
Can't this done with `sed` instead of pulling in the gettext-base package?